hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/impl/AssociatePrecompile.java

```diff
@@ -68,10 +75,22 @@ public AssociatePrecompile(
     @Override
     public TransactionBody.Builder body(final Bytes input, final UnaryOperator<byte[]> aliasResolver) {
         associateOp = decodeAssociation(input, aliasResolver);
+        accountId = Id.fromGrpcAccount(Objects.requireNonNull(associateOp).accountId());
```

Comment:
Q: though it's of accountID type contracts will still work right?

Reply:
My assumption is yes but... I'll have to see while writing the itests.